stat.proto

```diff
@@ -120,4 +121,6 @@ message StatJourneySection {
     optional string     to_admin_insee = 29;
 }
 
-
+message StatResponse {
```

Comment:
Not so convinced by that name, but I have nothing better to offer...
What other attributes do you imagine adding in this object?
Maybe Stat[Header|Common|General]Response (pick one), as opposed to existing StatJourneyResponse.
Because with StatResponse one might think having StatJourneyResponse contained in it.